acceptance-tests/src/test/java/tech/pegasys/web3signer/tests/bulkloading/AwsSecretsManagerAcceptanceTest.java

```diff
@@ -0,0 +1,129 @@
+import org.junit.jupiter.api.condition.EnabledIfEnvironmentVariable;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+@EnabledIfEnvironmentVariable(
+    named = "RW_AWS_ACCESS_KEY_ID",
+    matches = ".*",
+    disabledReason = "RW_AWS_ACCESS_KEY_ID env variable is required")
+@EnabledIfEnvironmentVariable(
+    named = "RW_AWS_SECRET_ACCESS_KEY",
+    matches = ".*",
+    disabledReason = "RW_AWS_SECRET_ACCESS_KEY env variable is required")
+@EnabledIfEnvironmentVariable(
+    named = "AWS_ACCESS_KEY_ID",
+    matches = ".*",
+    disabledReason = "AWS_ACCESS_KEY_ID env variable is required")
+@EnabledIfEnvironmentVariable(
+    named = "AWS_SECRET_ACCESS_KEY",
+    matches = ".*",
+    disabledReason = "AWS_SECRET_ACCESS_KEY env variable is required")
+@EnabledIfEnvironmentVariable(
+    named = "AWS_REGION",
+    matches = ".*",
+    disabledReason = "AWS_REGION env variable is required")
+@TestInstance(TestInstance.Lifecycle.PER_CLASS) // same instance is shared across test methods
+public class AwsSecretsManagerAcceptanceTest extends AcceptanceTestBase {
+  private static final String RW_AWS_ACCESS_KEY_ID = System.getenv("RW_AWS_ACCESS_KEY_ID");
+  private static final String RW_AWS_SECRET_ACCESS_KEY = System.getenv("RW_AWS_SECRET_ACCESS_KEY");
+  private static final String RO_AWS_ACCESS_KEY_ID = System.getenv("AWS_ACCESS_KEY_ID");
+  private static final String RO_AWS_SECRET_ACCESS_KEY = System.getenv("AWS_SECRET_ACCESS_KEY");
+  private static final String AWS_REGION =
+      Optional.ofNullable(System.getenv("AWS_REGION")).orElse("us-east-2");
+  private static final List<String> BLS_PRIVATE_KEYS =
+      List.of(
+          "3ee2224386c82ffea477e2adf28a2929f5c349165a4196158c7f3a2ecca40f35",
+          "32ae313afff2daa2ef7005a7f834bdf291855608fe82c24d30be6ac2017093a8");
+  private AwsSecretsManagerUtil awsSecretsManagerUtil;
+  private final List<String> awsKeysIdentifiers = new ArrayList<>();
+
+  @BeforeAll
+  void setupAwsResources() {
+    awsSecretsManagerUtil =
+        new AwsSecretsManagerUtil(AWS_REGION, RW_AWS_ACCESS_KEY_ID, RW_AWS_SECRET_ACCESS_KEY);
+
+    BLS_PRIVATE_KEYS.forEach(
+        key -> {
+          final String pubKey =
+              new BLSKeyPair(BLSSecretKey.fromBytes(Bytes32.fromHexString(key)))
+                  .getPublicKey()
+                  .toString();
+          awsSecretsManagerUtil.createSecret(pubKey, key);
+          awsKeysIdentifiers.add(pubKey);
+        });
+  }
+
+  @ParameterizedTest
+  @ValueSource(booleans = {true, false})
+  void secretsAreLoadedFromAWSSecretsManagerAndReportedByPublicApi(final boolean useConfigFile) {
+    final AwsSecretsManagerParameters awsSecretsManagerParameters =
+        new DefaultAwsSecretsManagerParameters(
+            AWS_REGION,
+            RO_AWS_ACCESS_KEY_ID,
+            RO_AWS_SECRET_ACCESS_KEY,
+            List.of(awsSecretsManagerUtil.getSecretsManagerPrefix()));
```

Comment:
What about tests for tag name and tag value prefix? Maybe we don't need these though as there are tests in signers?

Reply:
they are covered in signers ... I can still add them in web3signer though.

Reply:
added them as part of refactored AwsBulkLoaderArtifactSignerProvider class.